ui.py

```diff
@@ -230,3 +252,164 @@ def show(self) -> None:
+        --copilot-accept-background: var(--background);
+        --copilot-accept-border: var(--greenish);
+    }}
+
+    .{class_name} {{
+        margin: 1rem 0.5rem 0 0.5rem;
+    }}
+
+    .{class_name} .header {{
+        display: block;
+        margin-bottom: 1rem;
+    }}
+
+    .{class_name} a {{
+        border-radius: 3px;
+        border-style: solid;
+        border-width: 1px;
+        display: inline;
+        padding: 5px;
+        text-decoration: none;
+    }}
+
+    .{class_name} a.accept {{
+        background: var(--copilot-accept-background);
+        border-color: var(--copilot-accept-border);
+        color: var(--copilot-accept-foreground);
+    }}
+
+    .{class_name} a.accept i {{
+        color: var(--copilot-accept-border);
+    }}
+    """.format(
+        class_name=CSS_CLASS_NAME
+    )
+    COMPLETION_TEMPLATE = reformat(
+        """
+        <h4>Synthesizing {index}/{total_solutions} solutions (Duplicates hidden)</h4>
+        <hr>
+        {sections}
+        """
+    )
+    COMPLETION_SECTION_TEMPLATE = reformat(
+        """
+        <div class="header">{header_items}</div>
+        ```{lang}
+        {code}
+        ```
+        """
+    )
+
+    def __init__(self, view: sublime.View) -> None:
+        self.view = view
+        self.completion_manager = ViewCompletionManager(view)
+
+    @property
+    def completion_content(self) -> str:
+        syntax = self.view.syntax() or sublime.find_syntax_by_name("Plain Text")[0]
+        view_id = int(remove_prefix(get_copilot_view_setting(self.view, "panel_id", -1), "copilot://"))
+        completions = self._synthesize(self.completion_manager.panel_completions)
+        return self.COMPLETION_TEMPLATE.format(
+            index=len(self.completion_manager.panel_completions),
+            total_solutions=get_copilot_view_setting(self.view, "panel_completion_target_count", 0),
+            sections="\n\n<hr>\n".join(
+                self.COMPLETION_SECTION_TEMPLATE.format(
+                    header_items=" &nbsp;".join(self.completion_header_items(completion, view_id, index)),
+                    score=completion["score"],
+                    lang=basescope2languageid(syntax.scope),
+                    code=self._prepare_popup_code_display_text(completion["displayText"]),
+                )
+                for index, completion in enumerate(completions)
+            ),
+        )
+
+    @staticmethod
+    def completion_header_items(completion: CopilotPayloadPanelSolution, view_id: int, index: int) -> List[str]:
+        return [
+            """<a class="accept" href='{}'><i>✓</i> Accept</a>""".format(
+                sublime.command_url(
+                    "copilot_accept_panel_completion_shim",
+                    {"view_id": view_id, "completion_index": index},
+                )
+            ),
+            "(Mean Probability: {})".format(completion["score"]),
+        ]
+
+    def open(self) -> None:
+        # TODO: show this side-by-side?
```

Comment:
Probably, eventually, we could make this configurable

VSCode opens in a split window to the right

Reply:
Yeah, I think that should be in this PR tbh. Doesn't make sense to hide the view your working in. 

Btw I made you a contributor to this repo.